storage/ndb/rest-server2/server/src/batch_feature_store_ctrl.cpp

```diff
@@ -306,7 +308,11 @@ void BatchFeatureStoreCtrl::batch_featureStore(
                                     featureStatus);
   fsResp.features = features;
   fsResp.metadata = GetFeatureMetadata(metadata, reqStruct.metadataRequest);
-  resp->setContentTypeCode(drogon::CT_APPLICATION_JSON);
+  if (use_compressed) {
```

Comment:
use_compressed is true then CT_APPLICATION_OCTET_STREAM?
CT_APPLICATION_OCTET_STREAM is compressed isn't it

Reply:
CT_APPLICATION_OCTET_STREAM means it is application specific and that the web server should not bother with compressing it or other special actions. I made sure that the Content-Type in the protocol is still JSON.